docs/reference/user-management.md

```diff
@@ -0,0 +1,107 @@
+# User management
+
+User management in n8n allows you to invite people to work in your n8n instance. It includes:
+
+* Login and password management
+* Adding and removing users
+* Two account types: owner and member
+
+User management is available for self-hosted n8n. It isn't currently available for Cloud.
+
+## Setup
+
+There are three stages to set up user management in n8n:
+
+1. Configure your n8n instance to use your SMTP server.
+2. Start n8n and follow the setup steps in the app.
+3. Invite users.
+
+### Step one: SMTP
+
+You need an SMTP server for user management to send invites and password resets. Get the following information from your SMTP provider:
+
+* Server name
+* SMTP username
+* SMTP password
+* SMTP sender name
+
+To set up SMTP with n8n, configure the following environment variables for your n8n instance:
```

Comment:
Should have a link here to something that explains how to apply environment vars

Reply:
Added